k8s/templates/studio-deployment.yaml

```diff
@@ -60,8 +60,8 @@ spec:
             cpu: 0.5
             memory: 2Gi
           limits:
-            cpu: 0.5
-            memory: 2Gi
+            cpu: 2
+            memory: 8Gi
```

Comment:
I suggest we keep the frontend pod's limits the same as its requests. This is because in Kubernetes, if they're the same, it ensures that it never gets kicked out in case of memory or CPU pressure. Here's a link on k8s' QoS: https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed

Reply:
Thanks for this info! I just pushed the change. I think this does make a good argument for having less, but beefier, guaranteed pods then, as if we keep the limits small we increase the chances a pod will exceed them, and I don't know if we have a strong grasp of what limits we can safely stay under yet (as our publishing issues attest). We can revisit this next week though.

Reply:
That's true! I think we have a good argument for that side now. Let's add this as an entry in content-dev and discuss next week!